src/scripts/dfnshelper.js

```diff
@@ -0,0 +1,103 @@
+import * as locale from 'date-fns/locale'
+define(["globalize"], function (globalize) {

```

Comment:
```suggestion
import { ar, be, bg, ca, cs, da, de, el, enGB, enUS, es, faIR, fi, fr, frCA, he, hi, hr, hu, id, it, kk, ko, lt, ms, nb, nl, pl, ptBR, pt, ro, ru, sk, sl, sv, tr, uk, vi, zhCN, zhTW } from 'date-fns/locale';
import globalize from 'globalize';

```

Mixing ES6 imports and AMD imports is a bad idea. You can import an AMD module in ES6 in jellyfin-web, for now. We have a Babel plugin that will convert your imports to AMD.

Importing an entire package with `*` is also bad practice. We don't want to import thing we don't want and it's clearer as to what is used or not.

Reply:
> We have a Babel plugin that will convert your imports to AMD.

Do note that your IDE might complain that it can't find the module. This is normal for now, as we need to keep the AMD module paths the same. It'll be fixed in a final PR at the end of the migration.

Reply:
@MrTimscampi Indeed! Thank you very much! 

Although I commited this today, this was written way back before the ES6 migration started, between the breakage I made with the first attempt to use globalize locally (now moved to #1004).